transformations/concept2sentence/transformation.py

```diff
@@ -0,0 +1,92 @@
+import itertools
+import random
+
+from interfaces.SentenceOperation import SentenceAndTargetOperation
+from tasks.TaskTypes import TaskType
+
+from sibyl import Concept2Sentence
+
+
+class C2S(SentenceAndTargetOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION
+    ]
+    languages = ["en"]
+    keywords = [
+        # type of linguistic change
+        "lexical", 
+        "semantic",
+        # type of algorithm
+        "model-based", 
+        "transformer-based", 
+        "api-based", 
+        "tokenizer-required",
+        # naturalness of the generation
+        "unnaturally-written",
+        # potential accuracy & precision of the generation
+        "possible-meaning-alteration",
+        "low-precision",
+        "high-coverage",
+        "high-generations"
+    ]
+
+    def __init__(self, 
+                 seed=0, 
+                 max_outputs=1, 
+                 dataset=None, 
+                 extract="token",
+                 gen_beam_size=10,
+                 text_min_length=10,
+                 text_max_length=32,
+                 device='cuda',
+                 task_config=None):
+
+        super().__init__(seed, max_outputs=max_outputs)
+        self.seed = seed
+        self.max_outputs = max_outputs
+        self.c2s = Concept2Sentence(
+                        dataset = dataset,
+                        extract = extract,
+                        gen_beam_size = gen_beam_size,
+                        text_min_length = text_min_length,
+                        text_max_length = text_max_length,
+                        device = device)
+        self.task_config = task_config
+        if self.task_config is None:
+            self.task_config = {
+                'input_idx': [1],
+                'tran_type': 'INV',
+                'label_type': 'hard',
+                'task_name': 'topic'
+            }
+
+    def generate(self, sentence: str, target=None):
+        new_sentences, new_targets = [], []
+        for _ in range(self.max_outputs):
+            new_sentence, new_target = self.c2s.transform_Xy(sentence, target, self.task_config)
```

Comment:
Also you might want to mention the specific types of targets that "transform_Xy" expects.

Reply:
Added to the README and to the transformation method's docstring